### PR TITLE
Create Header component README.md

### LINKED ISSUE #912: Create Header component Readme.md
**Is your feature request related to a problem? Please describe.**
We currently don't have explanations of what each component does, how it should be used, how it shouldn't be used.

We should add these explanations for the Header component. 

This is a prerequisite to moving components to an external component directory. 

**Describe the solution you'd like**
We should have a `README.md` file in the `src/app/components/Header/` directory. 
It should have these sections.

```
- ## Description (high-level summary that will be accessible to product/UX/Test, plus link to Storybook, explain the different cases)
- ## When to use this component (including where it is currently used)
- ## When to not use this component (if this is not known, can leave the section empty)
- ## Accessibility notes (using accessiblity notes on Zeplin https://app.zeplin.io/project/5baa564c05c2eba0d2b89568/screen/5babb3dea83a3937890e374c, ensuring they make sense to non-technical audiences)
- ## Roadmap (what we have planned for this component)
- ## Additional notes (link to any relevant ADRs)
```

**Describe alternatives you've considered**
We have considered other sections in the readme, but these seem like a good starting point. We can iterate/add/change sections in future, if needed. 

**Testing notes**
The `README.md` file should be clear, understandable, to people who are not developers and give a good explanation about the scope of a component. 

**Additional context**
N/A

- [x] Initially labelled with ["Refinement needed"](https://github.com/BBC-News/simorgh/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by dr3**
Resolves #912

Adds a `README.md` file to the Header component with an explanation of what the component does, how it should be used, how it shouldn't be used.

- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: src/app/components/Header/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Header [![Storybook](https://github.com/storybooks/press/blob/master/badges/storybook.svg)](https://simorghstorybook.now.sh/?selectedKind=Header)
+
+### Description
+This component functions as a wrapper for the [Brand](../Brand) component which is also used in the [Footer](../Footer). The wrapper is a HTML 'header' element with the 'role' of 'banner' for accessibility. This is component is distinct from brand as the 'header' HTML element is unique and cannot be reused elsewhere on the page.
```
Comment (ChrisBAshton) [inline src/app/components/Header/README.md]:
```suggestion
This component functions as a wrapper for the [Brand](../Brand) component (which is also used in the [Footer](../Footer)). The wrapper is a HTML 'header' element with the 'role' of 'banner' for accessibility. This component is distinct from Brand as the 'header' HTML element is unique and cannot be reused elsewhere on the page.
```

File: src/app/components/Header/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+	
+### When to use this component
+Header is designed to be used at the top of the document above all other content.
+
+As this component uses the [Brand](../Brand) which consumes the service context via the [ServiceContextConsumer](../ServiceContextConsumer) component.
```
Comment (ChrisBAshton) [inline src/app/components/Header/README.md]: This sentence looks unfinished...?

File: src/app/components/Header/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+### Accessibility notes
+HTML: 
+Link (NB: Sits within header)
```
Comment (ChrisBAshton) [inline src/app/components/Header/README.md]: These two lines feel a bit redundant (what are they for?)

**Review by ChrisBAshton — CHANGES_REQUESTED** [review]
A typo, and then a couple of things to clarify I think. 👍

**Review by ChrisBAshton — APPROVED** [review]

**Review by jtart — APPROVED** [review]

**Comment by dr3** [comment]
Putting in test, just needs merging

**Comment by jamesbrumpton** [comment]
👍

**Pull request merged**